styx-scheduler-service/src/main/java/com/spotify/styx/flyte/FlyteAdminClientRunner.java

```diff
@@ -44,6 +44,8 @@
 public class FlyteAdminClientRunner implements FlyteRunner {
 
   private static final Logger LOG = LoggerFactory.getLogger(FlyteAdminClientRunner.class);
+  @VisibleForTesting static final String TERMINATE_CAUSE = "Styx halt";
```

Comment:
should we make this a bit more descriptive? I feel that it's not going to be clear that there was triggered a terminate event